SETUP/tests/jsTests/hieroTests.js

```diff
@@ -1,38 +1,38 @@
 /* global QUnit add */
 
-QUnit.module("Hiero tests", function(hooks) {
+QUnit.module("Hiero tests", function (hooks) {
     let hierobox;
-    hooks.beforeEach(function() {
-        hierobox = document.createElement('textarea');
-        hierobox.name = 'hierobox';
+    hooks.beforeEach(function () {
+        hierobox = document.createElement("textarea");
```

Comment:
This file and `tools/proofers/hiero/hiero.js` are going away with PR #1335.

Reply:
Yeah, one of us will have to resolve the conflict based on merge order. If I left this file out, the checks would fail as this file wouldn't match required format. I can help with that if necessary @cpeel .

Reply:
That's easy for me to fix in mine after this goes in, thanks for thinking of it though!